services/src/modules/directives/policy/types.ts

```diff
@@ -54,3 +54,8 @@ export type PolicyCacheKey = {
   metadata: ResourceMetadata;
   args?: PolicyArgsObject;
 };
+
+// Temporary until opa-wasm package adds types
+export interface LoadedPolicy {
+  evaluate: (input: { [key: string]: any }) => any[];
```

Comment:
```suggestion
  evaluate: (input: Record<string, unknown>) => { result: { allow: boolean } }[];
```

Reply:
The `LoadedPolicy` is supposed to be a generic type for a loaded WASM policy, to use until opa-wasm module is typed and replaced with the new type later.

I'll change it to `Record`, but the type of the result is a convention of our policy and not something for the generic loaded WASM policy type